sdk/servicebus/azure-servicebus/stress/scenarios-matrix.yaml

```diff
@@ -1,7 +1,32 @@
-displayNames:
 matrix:
   image:
     - Dockerfile
   scenarios:
-    sbStress:
-      testTarget: servicebus
\ No newline at end of file
+    queue:
+      testTarget: queue
+    aqueue:
+     testTarget: aqueue
+    queuepull:
+      testTarget: queuepull
+    aqueuepull:
+     testTarget: aqueuepull
+    batch:
+      testTarget: batch
+    abatch:
+      testTarget: abatch
+    queuew:
+      testTarget: queuew
+    aqueuew:
+     testTarget: aqueuew
+    queuepullw:
+      testTarget: queuepullw
+    aqueuepullw:
+     testTarget: aqueuepullw
+    batchw:
+      testTarget: batchw
+    abatchw:
+      testTarget: abatchw
+    memray:
+      testTarget: memray
```

Comment:
Thanks for putting in memray into the stress tests. I need to check with them if they have any suggestions on perhaps splitting the output file etc.